tests/unit/core/test_store.js

```diff
@@ -87,4 +107,20 @@ describe(__filename, () => {
     expect(typeof middleware({ _config, _window })).toBe('function');
     sinon.assert.notCalled(_window.__REDUX_DEVTOOLS_EXTENSION__);
   });
+
+  describe('minimalReduxLogger', () => {
+    it('process an action', () => {
```

Comment:
I don't really understand what's going on in this test either. I've read through it a few times, but there's always a point at which I get lost. :-(

Reply:
It simulates how Redux middleware works.

Reply:
Did the documentation help? It's a well defined interface so I don't think it will change.